files/routes/static.py

```diff
@@ -225,9 +225,6 @@ def log(v):
 
 	kind = request.values.get("kind")
 
-	if kind not in ACTIONTYPES:
-		logging.warning(f'Unfamiliar ACTIONTYPE {kind}')
-
```

Comment:
I think this should actually be left in; it's a good warning that something is missing.

Reply:
Removed deliberately, though think I described why more thoroughly in the other linked comment on the issue. That logging conditional in the route will, in the default case, almost always cause a warning when someone loads the modlog: `kind = None` unless someone explicitly selects a kind in the UI dropdown to filter the log by, and `None not in ACTIONTYPES => True`.

It's a request parameter, not something derived from the backend, and the route has logic to fail-safe to an unfiltered listing (a few lines down from there) if the request `kind` is invalid.

Would you prefer a rework of the logic such that the None case doesn't log but other invalid `kind` request params do?

Reply:
Aha! Got it, that was a misunderstanding on my part then. S'all good.